providers/ms365/resources/ms365.lr

```diff
@@ -84,6 +86,26 @@ private microsoft.tenant.settings @defaults("isAppAndServicesTrialEnabled isOffi
   isOfficeStoreEnabled bool
 }
 
+// Company-wide settings for Microsoft Forms
+private microsoft.tenant.formsSettings @defaults("id isExternalSendFormEnabled isBingImageSearchEnabled") {
```

Comment:
please don't add the `id` to the defaults. That should (almost) never be in the defaults